pyproject.toml

```diff
@@ -196,5 +196,7 @@ filterwarnings = [
     # papermill is also using a deprecated method of getting current time
     'ignore:.*is deprecated and scheduled for removal.*:DeprecationWarning',
     # lightkurve is using a deprecated numpy interface
-    'ignore:.*numpy.core.einsumfunc is deprecated and has been renamed.*:DeprecationWarning'
+    'ignore:.*numpy.core.einsumfunc is deprecated and has been renamed.*:DeprecationWarning',
+    # Sometimes gaussian fits are not great
```

Comment:
A little hesitant to ignore the warning about the bad fit without feedback to the user of some sort.

Reply:
These warnings are only ignored during tests; when the code is running as normal this warning would be raised.